src/main/java/me/botsko/prism/listeners/PrismPlayerEvents.java

```diff
@@ -324,9 +324,15 @@ public void onPlayerInteract(final PlayerInteractEvent event) {
             // " Item in hand:" + player.getItemInHand().getTypeId() + ":" +
             // player.getItemInHand().getDurability());
 
+            int itemInHandId = player.getItemInHand().getTypeId();
```

Comment:
Oh, now I understand.  Your comment seemed to highlight specifically that one line, which is what caused the misunderstanding.

In newer versions of Bukkit/Spigot (for sure in 1.11, I haven't tested 1.10) the damage value of the ItemStack representing an empty hand is 0, instead of -1 in older versions.